### DISCUSSION TITLE
How do i use it.

### DISCUSSION THREAD

**Discussion by Nojarosann** [haven1433/HexManiacAdvance#86] [Q&A] [answered]
i litterally just downloaded it because i saw on [this](https://www.reddit.com/r/PokemonROMhacks/comments/322vrt/how_to_make_tms_with_unlimited_use_in_pokemon/) reddit post and i wanted to do it but i have no idea what a hex editor is is googled it on google and this is what came to me.

I have basically three questions
1 - Is there supposed to be a .exe in the folder after i extract it?
2 - Is there a video teaching how to use it or is it okay to use the wiki?

**Comment by haven1433** [comment, marked as answer]
Hi, thanks for your interest in the project!

1 - Each [Release](https://github.com/haven1433/HexManiacAdvance/releases) has multiple 'assets' associated with it. Make sure you download the actual program, not the source code. You're likely interested in [Version 0.4.3](https://github.com/haven1433/HexManiacAdvance/releases/download/v0.4.3/HexManiacAdvance0.4.3.zip)

2 - Here are some [Tutorials](https://github.com/haven1433/HexManiacAdvance/wiki/Tutorials), some of which are videos. You can also join the Discord server to get help using the application.

**Reply by Nojarosann** [reply]
Thanks!! Google is what recommended this to me so you're probably very lucky that your project is the first result for "Pokemon emerald hex editor" hope to see your project grow :)
